test/integration/model/schema.test.js

```diff
@@ -0,0 +1,458 @@
+            return restaurauntModel.save();
+          }).then(function() {
+            restaurauntModel = self.RestaurantTwo.build({bar: 'two.3'});
+            return restaurauntModel.save();
+          }).then(function() {
+            return self.RestaurantOne.findAll({
+              where: {bar: {$like: 'one%'}}
+            });
+          }).then(function(restaurantsOne) {
+            expect(restaurantsOne).to.not.be.null;
+            expect(restaurantsOne.length).to.equal(2);
+            restaurantsOne.forEach(function(restaurant) {
+              expect(restaurant.bar).to.contain('one');
+            });
+            return self.RestaurantOne.count();
+          }).then(function(count) {
+            expect(count).to.not.be.null;
+            expect(count).to.equal(2);
+            return self.RestaurantTwo.findAll({
+              where: {bar: {$like: 'two%'}}
+            });
+          }).then(function(restaurantsTwo) {
+            expect(restaurantsTwo).to.not.be.null;
+            expect(restaurantsTwo.length).to.equal(3);
+            restaurantsTwo.forEach(function(restaurant) {
+              expect(restaurant.bar).to.contain('two');
+            });
+            return self.RestaurantTwo.count();
+          }).then(function(count) {
+            expect(count).to.not.be.null;
+            expect(count).to.equal(3);
+          });
+        });
+      });
+
+
+      describe('Get associated data in public schema via include', function() {
+        beforeEach(function() {
+          var Location = this.Location;
+
+          return Location.sync({force: true})
+            .then(function() {
+              return Location.create({name: 'HQ'}).then(function() {
+                return Location.findOne({where: {name: 'HQ'}}).then(function(obj) {
+                  expect(obj).to.not.be.null;
+                  expect(obj.name).to.equal('HQ');
+                  locationId = obj.id;
+                });
+              });
+            })
+            .catch(function(err) {
+              expect(err).to.be.null;
+            });
+        });
+
+        it('should be able to insert and retrieve associated data into the table in schema_one', function() {
+          var self = this;
+
+          return self.RestaurantOne.create({
+            foo: 'one',
+            location_id: locationId
+          }).then(function() {
+            return self.RestaurantOne.findOne({
+              where: {foo: 'one'}, include: [{
+                model: self.Location, as: 'location'
+              }]
+            });
+          }).then(function(obj) {
+            expect(obj).to.not.be.null;
+            expect(obj.foo).to.equal('one');
+            expect(obj.location).to.not.be.null;
+            expect(obj.location.name).to.equal('HQ');
+          });
+        });
+
+
+        it('should be able to insert and retrieve associated data into the table in schema_two', function() {
```

Comment:
This test doesn't add anything new in regards to schemas  - the result is the same regardless of which schema you are using